lib/phoenix_live_view/html_tokenizer.ex

```diff
@@ -553,7 +553,7 @@ defmodule Phoenix.LiveView.HTMLTokenizer do
 
     meta =
       if context = get_context(context) do
-        Map.put(meta, :context, context)
+        Map.put(meta, :context, Enum.uniq(context))
```

Comment:
I am worried this will lead to side-effects. For example, this:

    [:comment_start, :comment_end, :comment_start]

will become:

    [:comment_start, :comment_end]

And that's innacurate. Perhaps it could be a test too?

Instead, you could do this:

```elixir
defp trim_context([:comment_start, :comment_end | rest]), do: trim_context(rest)
defp trim_context(rest), do: rest
```

WDYT?

Reply:
Done!. We also need to pattern match the list otherwise it would clean when they are not duplicated. Let  me know if that doesn't make sense 🙌🏻